modules/ROOT/pages/_partials/usage-reports-tasks.adoc

```diff
@@ -0,0 +1,50 @@
+// tag::accessUsageReports[]
+
+. Sign in to Anypoint Platform using an account that has the Usage Viewer permission in your root organization.
+. In the Anypoint Platform navigation bar, click the business group selector, and ensure you are at the root organization level.
+. You can access usage reports either by:
+* Clicking your username icon and then clicking *Usage*, or
+* Selecting *Usage* in the Anypoint Platform navigation menu.
+. Select the product for which you want to view a usage report. 
```

Comment:
"for which"

Reply:
@hannanelson  Select the product to view the usage report for.

?